<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailBar</class>
 <widget class="QWidget" name="DetailBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="winTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>XXXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>711</width>
     <height>411</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>709</width>
      <height>409</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#scrollAreaContents{
    background:transparent;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="allPay_lab">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>500</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
   color: #ffffff; /* 设置文本颜色为白色 */
   font-size: 20px; /* 设置字体大小 */
   padding: 20px; /* 设置内边距 */
}
</string>
   </property>
   <property name="text">
    <string>总计金额：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_ptn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>500</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color: #a18bf7; /* 设置背景颜色为淡紫色 */
   color: #ffffff; /* 设置文字颜色为白色 */
   border: none; /* 去除边框 */
  border-radius: 5px; /* 设置边框为5像素的圆角 */

}

   /* 悬停样式 */
   QPushButton:hover {
       background-color: #8570e8; /* 设置悬停时的背景颜色 */
   }

   /* 按下样式 */
   QPushButton:pressed {
       background-color: #6b57cc; /* 设置按下时的背景颜色 */
   }</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
